coverage/index.html

```diff
@@ -0,0 +1,111 @@
+
```

Comment:
the `coverage` folder should be added in `.gitignore` file

Reply:
Means these codes should not be uploaded to github

Reply:
yes,solved